plugins/arDominionB5Plugin/modules/accession/templates/_alternativeIdentifiers.php

```diff
@@ -22,13 +22,13 @@ class="collapse<?php echo 0 < count($alternativeIdentifierData) ? ' show' : '';
     <table class="table table-bordered mb-0 multi-row">
       <thead class="table-light">
         <tr>
-          <th id="alt-identifiers-type-head" style="width: 30%">
+          <th id="alt-identifiers-type-head" class="w-30">
```

Comment:
Nice work @sbreker and @FitiaHarena, just a comment about the widths in tables.

Bootstrap 5 only comes with a few sizing classes (25, 50, 75, 100), all the other width classes used in this PR should be created in the SASS code. This could be done using the utilities API from B5 or adding those classes manually.

https://getbootstrap.com/docs/5.0/utilities/sizing/

Another option would be to remove those fixed widths and leave the tables adapt to the content, as they should be responsible tables in B5 themes.